### PR TITLE
Some QoL changes

### PR DESCRIPTION
**Pull request by QtKaii**

### CODE REVIEW

**Comment by QtKaii** [comment]
???
There is no hardcoded secret in that file???

**Comment by QtKaii** [comment]
@Tolfx

File: src/Server/Routes/v2/Customers/Customers.config.ts [hunk 1]
```diff
@@ -669,3 +664,17 @@ export = class CustomerRouter
     }
 
 }
+
+async function passwordResetChecks(passwordReset: any, res: any)
```
Comment (Tolfx) [inline src/Server/Routes/v2/Customers/Customers.config.ts:668]: Don't use type `any`

**Review by Tolfx — CHANGES_REQUESTED** [review]
Fix typing errors.

**Comment by Tolfx** [comment]
> ??? There is no hardcoded secret in that file???

The testing files has tokens, but only used for testing so nothing to worry about.

**Pull request merged**